### PR TITLE
feat : add email type in validate-id

### PR DESCRIPTION
**Pull request by maksha19**
This PR adds... 
- add email type in validate-id

- [x] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
- [x] I've requested reviews from 1-2 reviewers
- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
- [x] I've added/updated unit/integration tests
- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
- [x] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
- [ ] I've added documentation in README/[Notion](https://www.notion.so/82b92fb1007640328dab9582c0a3694e?v=3b6ce48202cf40ad8450553799b13146)

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -257,3 +257,9 @@ export const DailyStatisticsResult = t.type({
 
 export type DailyStatistics = t.TypeOf<typeof DailyStatistics>;
 export type DailyStatisticsResult = t.TypeOf<typeof DailyStatisticsResult>;
+
+const EmailRecord = t.type({
+  emailAddress: t.string,
+});
+
+export type EmailRecord = t.TypeOf<typeof EmailRecord>;
```
Comment (szenius) [inline src/types.ts:265]: Eh i think there is a misunderstanding, the task was to add "EMAIL" as a validation type to features

**Review by szenius — CHANGES_REQUESTED** [review]

**Review by szenius — CHANGES_REQUESTED** [review]
Tested in mobile-application. Right now when the ID is invalid there is a dialog saying "System error". To show the correct errors we also need to add email validation in frontend. See `src/utils/validateIdentification.ts`

File: package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "react": "16.11.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz",
     "react-native-gesture-handler": "~1.8.0",
+    "react-native-joi": "0.0.5",
```
Comment (szenius) [inline package.json:39]: Why do we need this? Cannot use joi like in the backend?

Comment (maksha19) [reply package.json:39]: the backend `joi` library is not compatible for mobile application.

Comment (maksha19) [reply package.json:39]: File

Comment (szenius) [reply package.json:39]: Oic, there is a @hapi/joi in our dependencies, can you check if that is used? Else can remove

Comment (maksha19) [reply package.json:39]: `@hapi/joi` was moved to standalone `joi` package. which is not compatible for front-end.

Comment (szenius) [reply package.json:39]: What I mean is in `package.json` in this repo there is @hapi/joi, if we are not using that, please help to remove it. Thank you!

Comment (maksha19) [reply package.json:39]: @hapi/joi is not there in package.json. plz check it.

Comment (szenius) [reply package.json:39]: Ah my bad, it is in the lock file, not package.json

**Review by szenius — APPROVED** [review]
Great work!!! 🎉

**Pull request merged**